lib/scrivener/html.ex

```diff
@@ -233,11 +242,17 @@ defmodule Scrivener.HTML do
   defp li_classes_for_style(paginator, page_number, :semantic) do
     if(paginator.page_number == page_number, do: ["active", "item"], else: ["item"])
   end
+  defp li_classes_for_style(_paginator, :ellipsis, :materialize), do: []
+  defp li_classes_for_style(paginator, page_number, :materialize) do
+    if(paginator.page_number == page_number, do: ["active"], else: ["waves-effect"])
```

Comment:
Well, I'd like, but I'm not sure what's the best way to change it as a general (not only materialize) solution to add additional classes to elements.

I've tested your solution about icons, it works perfectly and it's very cool, thanks. I've added an example to the README